modules/mysql/src/main/java/org/testcontainers/containers/MySQLContainer.java

```diff
@@ -20,7 +20,7 @@ public class MySQLContainer<SELF extends MySQLContainer<SELF>> extends JdbcDatab
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("mysql");
 
     @Deprecated
-    public static final String DEFAULT_TAG = "5.7.34";
+    public static final String DEFAULT_TAG = "5.7.44";
```

Comment:
Any issue is we keep 5.7.34 as a default? I would like to avoid breaking existing test that relies on default constructor.

Reply:
I don't see why a patch/bugfix release would be expected to break things for people and if you check the history it was previously changed to a new patch releases.

Reply:
For what it's worth here's an example: 
https://github.com/testcontainers/testcontainers-java/commit/3aa5fbfa17be09f4e42b2300396890f76f508c86#diff-2568f90aa8a2109ff614414c5ef2b547f6bd7e8b8d4db9e81130d091b3e5d67cR19